docs/modules/ROOT/pages/index.adoc

```diff
@@ -1215,6 +1215,186 @@ In each of the above sections, the following changes MUST be applied:
 * The validator SHALL maintain one or more _<<_trust_list,trust lists>>_ which do not need to be the same as the _<<_trust_list,trust lists>>_ used to validate claim generator signatures as described in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_c2pa_signers++[Section 14.4.1, “C2PA Signers.”]
 * The validator SHALL consider any v1 time-stamp as defined in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_choosing_the_payload++[Section 10.3.2.5.2, “Choosing the payload”] to be invalid.
 
+== HTTPS API
+
+An _<<_actor,actor>>_ MAY wish to provide a service API for providing *<<_identity_assertion,identity assertion>>* signatures and credentials. This MAY be done via any protocol. This standard defines a RECOMMENDED API which can be made available via link:https://datatracker.ietf.org/doc/html/rfc9112[HTTP 1.1 (RFC 9112)] or any subsequent version of the HTTP protocol.
+
+The intent of this API is that a software library can be configured by providing a single HTTPS endpoint address. All remaining configuration can be discovered via that endpoint.
+
+This standard describes three endpoints:
+
+* *Configuration and discovery:* Provides additional information required for a client process to perform the remaining operations.
+* *User-specific options:* Provides additional information about an authenticated user which can be used to configure subsequent signing requests.
+* *Generate asset-specific credential:* Given a description of a specific _<<C2PA asset>>_ and a specific authenticated user, generate an appropriate `signature` value binding the user to the asset.
+
+The interaction between _<<_identity_assertion_generator,identity assertion generator>>_ and the service API is depicted in the following sequence diagram, which is non-normative:
+
+[mermaid,width=100%]
+....
+sequenceDiagram
+    participant IAG as Identity assertion generator
+    participant API as Service API
+
+    IAG->>API: Request configuration information
+    API->>IAG: Respond with configuration information
+    Note right of IAG: Caching of this information is encouraged
+
+    IAG->>API: Request user-specific options
+    API->>IAG: Respond with user-specific options
+    Note right of IAG: This may inform user experience<br>such as which verified<br>identities are available
+
+    IAG->>IAG: Generate signer_payload
+    IAG->>API: Request signature for user and signer_payload
+    API->>IAG: Generate signature
+
+    IAG->>IAG: Embed signature in identity assertion
+....
+
+=== Configuration and discovery
+
+This request is an HTTPS `GET` with no parameters.
+
+The endpoint address is communicated out-of-band from the API service provider to the _<<_identity_assertion_generator,identity assertion generator>>._
+
+This request MAY require authentication via a `Bearer` token.
```

Comment:
Is mentioning Bearer token limiting? Perhaps the user is not yet authenticated and an auth flow could be started at this point. Perhaps this would require stronger auth. Maybe just say that it "MAY require authentication"